configure.ac

```diff
@@ -173,6 +183,44 @@ AS_IF([test "$want_pi" = yes], [
     PI_url=https://github.com/p4lang/PI/
     AC_CHECK_HEADERS([PI/pi.h PI/target/pi_imp.h PI/p4info.h], [],
                      [AC_MSG_ERROR([Cannot find PI headers, did you install $PI_url])])
+
```

Comment:
I think we should use `AC_CONFIG_SUBDIRS`. 

```
AS_IF([test "$want_pi" = yes], [
  AC_CONFIG_SUBDIRS([targets/simple_switch_grpc])
])
```

This requires fewer changes and leaves all the simple_switch_grpc specific stuff in a separate configure file, without adding complexity for users.